### PR TITLE
Add UG fixes for PED issues

### PR DESCRIPTION
**Pull request by JasonYapzx**
# Status

COMPLETED

## Description
Added some user guide explanations on certain PED issues that were found. Please see the following issues:
https://github.com/AY2223S1-CS2103T-W17-2/tp/issues/130 : Multiple flag input
https://github.com/AY2223S1-CS2103T-W17-2/tp/issues/124 : Additional input for 1 word commands (`summary`, `help`, `exit`)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -181,6 +181,16 @@ For example, given the command `add t/e d/Lunch a/15.60 da/15-08-2022 c/Food`, l
 - Letters followed by a forward slash is called a prefix and will be used to specify a specific parameter<br>
   e.g `t/ ` refers to the entry type parameter.
 
+- PennyWise will always take the **LAST OCCURRENCE** of a command prefix should there be more than 1 of the same command
+  prefix in a given command. 
+  e.g In `add t/i d/Monthly Allowance a/200.00 da/01-10-2022 c/Allowance d/Oct Allowance d/October Allowance`, PennyWise
```
Comment (sharmaine1028) [inline docs/UserGuide.md:186]: Minor typo of e.g when it should be e.g. seems to be an inconsistency throughout...

Comment (sharmaine1028) [inline docs/UserGuide.md:186]: For this do you think its better if its in a block code instead?

**Review by sharmaine1028 — COMMENTED** [review]
Looks good except for some minor bugs

File: docs/UserGuide.md (hunk 1 above)
Comment (JasonYapzx) [reply docs/UserGuide.md:186]: Have fixed all the e.g.

Comment (JasonYapzx) [reply docs/UserGuide.md]: yeap have refactored it to show a block code, does look a bit neater

File: docs/UserGuide.md [hunk 2]
```diff
@@ -179,7 +179,26 @@ For example, given the command `add t/e d/Lunch a/15.60 da/15-08-2022 c/Food`, l
...
+  You should expect to see this output:
+    <pre>
+    <code>
+        New entry added: October Allowance; Date: 01-10-2022; Amount: 200.00; Tag: Allowance.
+    </code>
+  </pre>
+
+- PennyWise will **IGNORE** any extra words or characters present after the following commands: 
+ `summary` (except [`summary [mo/MONTH]`](#summary-of-entries)) `help`, `clear` and `exit`. 
+  e.g. `summary abcdefg` will be interpreted as `summary` or `help 123456` will also be interpreted as `help`.
```
Comment (EmilyOng) [inline docs/UserGuide.md:201]:
```suggestion
  e.g. `summary abcdefg` will be interpreted as `summary` and `help 123456` will also be interpreted as `help`.  
```

**Review by EmilyOng — APPROVED** [review]
LGTM!

**Review by ruihan00 — COMMENTED** [review]
LGTM

**Pull request merged**